os/android/drmhwctwo.cpp

```diff
@@ -264,16 +265,28 @@ HWC2::Error DrmHwcTwo::HwcDisplay::GetChangedCompositionTypes(
   return HWC2::Error::None;
 }
 
-HWC2::Error DrmHwcTwo::HwcDisplay::GetClientTargetSupport(uint32_t /*width*/,
-                                                          uint32_t /*height*/,
-                                                          int32_t /*format*/,
+HWC2::Error DrmHwcTwo::HwcDisplay::GetClientTargetSupport(uint32_t width,
+                                                          uint32_t height,
+                                                          int32_t format,
                                                           int32_t dataspace) {
-  if (dataspace != HAL_DATASPACE_UNKNOWN &&
-      dataspace != HAL_DATASPACE_STANDARD_UNSPECIFIED)
+  if (width != display_->Width() || height != display_->Height()) {
     return HWC2::Error::Unsupported;
+  }
 
-  // TODO: Validate format can be handled by either GL or planes
-  return HWC2::Error::None;
+  if (format == HAL_PIXEL_FORMAT_RGBA_8888 &&
+      (dataspace == HAL_DATASPACE_UNKNOWN ||
+       dataspace == HAL_DATASPACE_STANDARD_UNSPECIFIED)) {
+    return HWC2::Error::None;
+  } else {
+    // Convert HAL to fourcc-based DRM formats
+    uint32_t format = convert_format(format);
```

Comment:
hmm u want to rename this local variable as drm_format.
convert_format -> GetDrmFormat( ?